common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java

```diff
@@ -41,9 +45,12 @@ public static void setSystemProperties(Configuration configuration, boolean isAg
 
     if (!configuration.spanEndpoint.toLowerCase().startsWith("http://")
         && !configuration.spanEndpoint.toLowerCase().startsWith("https://")) {
-      String msg = "Invalid configuration: span endpoint should start with http:// or https://.";
-      logger.severe(msg);
-      throw new IllegalStateException(msg);
+
```

Comment:
Can we add a comment here to point out this is something done to keep backwards compatibility?

Reply:
added